tailwind.config.cjs

```diff
@@ -12,7 +12,14 @@ const config = {
 		}
 	},
 
-	plugins: [require('flowbite/plugin')],
+	plugins: [
+		require('flowbite/plugin'),
+		({ addBase }) => {
+			addBase({
+				'a': { color: 'var(--link-color)' }
```

Comment:
Gooi nou geen globale css in een willekeurig bestand, helemaal kut om te debuggen.

+ je hebt hiermee strafbakken gesloopt
![afbeelding](https://github.com/ODDInvictus/ibs3/assets/56843328/75e4df94-1eee-4797-983f-c1297d27529e)

Reply:
om een of andere reden zorgt de tailwind reset ervoor dat a { color: } niet werkt, dus dit is idd de enige oplossing

ik heb wel even een comment gegooid in typography.css

Reply:
ooit sloop ik heel tailwind eruit